core/src/com/unciv/ui/utils/LimitOrientationsHelper.kt

```diff
@@ -8,12 +8,6 @@ import com.unciv.models.metadata.GameSettings
  *  position changes to automatically result in App orientation changes. This is about limiting that feature.
  */
 interface LimitOrientationsHelper {
-    /** Set a specific requested orientation or pull the setting from disk and act accordingly
-     *  @param newOrientation A SCREEN_ORIENTATION_* value from [ActivityInfo]
-     *          or SCREEN_ORIENTATION_UNSPECIFIED to load the setting
-     */
-    fun limitOrientations(newOrientation: Int)
-
```

Comment:
Does this interface still make sense if we remove one of the only two functions in it?

Reply:
That's still the platform-specific / platform-independent boundary pattern. The interface is the platform-agnostic common ground, even if no other platform implements it.

Hmm... There were tiltable monitors for PC's and some communication and API's so the desktop / running software could react. We don't need to cater for that as it would be seen as resize events already. But imagine those had a motor and an API for that so software could _set_ its orientation... :rofl: Never leave a full coffee mug near a tilt monitor. Even a non-motorized one.